.github/workflows/check-openapi-changes.yaml

```diff
@@ -0,0 +1,122 @@
+# ℹ️
+# This workflow runs for pull_requests that are base on main
+# if there is a change in the file `openapi.yaml`
+# but only when opening a PR with changes to the file
+# of if a commit is added to a PR with changes to the file
+#
+#    - 1. it runs the coral setup
+#    - 2. auto generates the file `coral/types/api.t.ds` from the `openapi.yaml`
+#    - 3. checks if the file file `coral/types/api.t.ds` was changed
+#       - if not: ends the workflow ✅
+#       - if yes:
+#         - 3.1 runs the typescript compiler in 'coral/'
+#         - 3.2 runs the frontend tests in 'coral/'
```

Comment:
First, I just removed the test run, because we don't really need it here :D and amended the updated comment (because I tried to move the comment in a `description` key and update it, but that failed). That's why 🙈